Modules/Timer/Timer/StartEdit/StartEditReducer.swift

```diff
@@ -24,6 +25,28 @@ func createStartEditReducer(repository: TimeLogRepository, time: Time) -> Reduce
         case .doneButtonTapped:
             return doneButtonTapped(state, repository)
 
+        case .stopButtonTapped:
+            guard state.editableTimeEntry != nil else {
```

Comment:
With something like this you will avoid all the force unwrapping that follow:
```suggestion
            guard let editableTimeEntry = state.editableTimeEntry else {
```